### ISSUE TITLE
Pyload yet again :o

### ISSUE THREAD

**Issue by IanJF** [pyload/pyload#2774]
Hi,

A few days ago, Pyload on my QNAP stopped working. The Download option seemed to be automatically turned off. I did what I did before to get Pyload working from advice on here (which I probably shouldn't off as it's a different issue slightly). I replaced the UpdateManager.py - tried many reinstalls but nothing worked... Oh it fixed the Download Off issue, but I was back to the no premium account problem for UploadedTo. The account wouldn't register as Premium.

I tried installing and uninstalling many times and copying the UpdateManger.py along with deleting the default userplugins/ folder etc. Nothing seemed to work, UploadedTo would not register as Premium.

Anyway I saw a few other posts and followed instructions on those. One was to issue a command like this:

"python /opt/QPyload/pyLoadCore.py --configdir=/opt/QPyload/CONFIG -u "

I pointed it where my pyLoadCore.py file was. It went through a few things and then said there was a file or a folder missing.
However, after doing this I seem to have completely broken Pyload. Now when I start up Pyload I just get an error saying: "Critical Error while processing Request". I've tried uninstalling, reinstalling, replacing UpdateManager.py etc. But all I get is this same error page.

Any help to get this going again would be very much appreciated.

Best Regards,
Ian.

**Comment by GammaC0de** [comment]
Most of the issues you were describing were already been fixed already.
What I would do is delete /opt/QPyload/ and then reinstall pyload.

**Comment by IanJF** [comment]
Thanks GammaC0de - reinstalling wouldn't work, I think it was an issue with QNAP. I had to restart the QNAP itself, then reinstall Pyload. That seemed to do the trick. I deleted the /userplugins folder, replaced the UpdateManager.py files and restarted Pyload.
Pyload started fine - The Download option was turned on - did not have the "Update available" message displayed... I added my UploadedTo user account... unfortunately it still doesn't show as Premium.

I guess I am nearly there... just can't get the Premium Account to work which I guess is the most important part :)

Thanks,
Ian.
